src/Compilers/CSharp/Portable/Lowering/Instrumentation/DynamicAnalysisInjector.cs

```diff
@@ -254,16 +261,16 @@ private Cci.DebugSourceDocument GetSourceDocument(CSharpSyntaxNode syntax, FileL
             return _debugDocumentProvider.Invoke(path, basePath: "");
         }
 
-        private BoundStatement AddAnalysisPoint(CSharpSyntaxNode syntaxForSpan, SyntheticBoundNodeFactory statementFactory)
+        private BoundStatement AddAnalysisPoint(CSharpSyntaxNode syntaxForSpan, Text.TextSpan? alternateStartSpan, SyntheticBoundNodeFactory statementFactory)
```

Comment:
This was just the easiest thing to code--syntax nodes have spans, and the alternate place to start is derived from a syntax node--but I'm happy to use whatever representation you prefer.